src/Access/Common/AuthenticationData.cpp

```diff
@@ -85,6 +95,40 @@ AuthenticationData::Digest AuthenticationData::Util::encodeSHA256(std::string_vi
 #endif
 }
 
+AuthenticationData::Digest AuthenticationData::Util::encodeBcrypt(std::string_view text [[maybe_unused]])
+{
+#if USE_BCRYPT
+    char salt[BCRYPT_HASHSIZE];
+    Digest hash;
+    hash.resize(64);
+    int ret = bcrypt_gensalt(12, salt);
```

Comment:
@evillique , as mentioned in slack, Bcrypt password itself contains the work factor (set to 12 here) and the salt. The work factor determine how many iterations (2^workfactor) and needed to change over time. For example, right now OWASP recommends a number bigger than 10 but that could change in the future.